### PR TITLE
Feature/is fraction function master

### PR DESCRIPTION
**Pull request by shaharkazaz**

### CODE REVIEW

File: utils/src/number/index.ts [hunk 1]
```diff
@@ -43,4 +43,16 @@ export function random(lower: number = 0, upper: number = 1) {
         return Math.random();
     }
     return lower + Math.floor(Math.random() * (upper - lower + 1));
+}
+
+/**
+ * Checks if a given number is a fraction or an integer
```
Comment (NetanelBasal) [inline utils/src/number/index.ts]: add tests, please.

File: utils/src/number/index.ts [hunk 2]
```diff
@@ -43,4 +43,16 @@ export function random(lower: number = 0, upper: number = 1) {
...
+ * @param {number} number - The number to check
+ * @returns {boolean} - True if the number is a fraction
+ */
+export function isFraction(number: number) {
+    if (!isNumber(number)) {
+        throw 'value to check is not a number';
```
Comment (NetanelBasal) [inline utils/src/number/index.ts]: `throw `expected a number but got ${number}`;`

Comment (shaharkazaz) [reply utils/src/number/index.ts]: maybe ${typeof number}?

Comment (NetanelBasal) [reply utils/src/number/index.ts]: 🔥

**Comment by shaharkazaz** [comment]
@NetanelBasal Pushed the fixes, tests + error msg

**Comment by NetanelBasal** [comment]
@shaharkazaz well done.

**Pull request merged**